### PR TITLE
[Docs] Update change log with open issues and limitations

### PR DESCRIPTION
**Pull request by jillnogold**
Also 2 doc fixes: [ML-4166](https://jira.iguazeng.com/browse/ML-4166), [ML-4166](https://jira.iguazeng.com/browse/ML-4166)

### CODE REVIEW

File: docs/change-log/index.md [hunk 1]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
-|ML-2407| Kafka ingestion service on an empty feature set returns an error.      |Ingest a sample of the data manually. This creates the schema for the feature set and then the ingestion service accepts new records.|v1.1.0    |
-|ML-2489| Cannot pickle a class inside an mlrun function.                       |Use cloudpickle instead of pickle.|v1.2.0    |
-| [2621](https://github.com/mlrun/mlrun/issues/2621)| Running a workflow whose project has `init_git=True`, results in Project error|Run `git config --global --add safe.directory '*'` (can substitute specific directory for *).                                                                                                                                                                                                                                                                                    |v1.1.0    |
+|ML-1373|When attempting to ingest data with RemoteSpark using an incorrect service name, the pods get stuck during ContainerCreating. | NA | v0.9.0 |
```
Comment (gilad-shaham) [inline docs/change-log/index.md:720]: workaround: verify the correct service name

File: docs/change-log/index.md [hunk 2]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-3445|`project.deploy_function` operation might get stuck when running v1.3.0 demos on an Iguazio platform running v3.2.x.| Replace code: `serving_fn = mlrun.new_function("serving", image="python:3.9", kind="serving", requirements=["mlrun[complete]", "scikit-learn~=1.2.0"])` with: <br>`function = mlrun.new_function("serving", image="python:3.9", kind="serving") function.with_commands([ "python -m pip install --upgrade pip", "pip install 'mlrun[complete]' scikit-learn==1.1.2", ])`|v1.3.0    |
+|ML-3480|Documentation: request details on label parameter of feature set definition.| NA | v1.2.1    |
+|NA|The feature store does not support schema evolution and does not have schema enforcement.| NA| v1.2.1    |
+|ML-3633|Fail to import a context from dict                                     |When loading a context from dict (e.g.: mlrun.MLClientCtx.from_dict(context)), make sure to provide datetime objects and not string. Do this by executing `context['status']['start_time'] = parser.parse(context['status']['start_time'])<br> context['status']['last_update'] = parser.parse(context['status']['last_update'])` prior to `mlrun.MLClientCtx.from_dict(context)`|v1.3.0    |
```
Comment (gilad-shaham) [inline docs/change-log/index.md:736]: The description here is too generic, need better explanation

File: docs/change-log/index.md [hunk 3]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-3640|When running a remote function/workflow, the `context` global parameter is not automatically injected.| Use `get_or_create_ctx`| v1.3.0    |
+|ML-3714|It can happen that an MLrun pod succeeds but there's an error in Kubeflow. | NA | v1.3.0 |
+|ML-4153|When creating a passthrough features-set in the UI, with no online target, the feature-set yaml includes a parquet offline target, which is ignored.| NA | v1.4.0  |
+|ML-4370|
```
Comment (gilad-shaham) [inline docs/change-log/index.md:740]: Missing info

File: docs/change-log/index.md [hunk 4]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-4740|When running function `batch_inference_v2` from the SDK, the `ingest()` function accepts 3 parameters as Data-item or other types: `dataset`, `model_path` and `model_endpoint_sample_set`. If you provided these parameters as non Data-items and later on you want to rerun this function from the UI, you need to provide these parameters as Data-item.|Prepare suitable Data-item and provide it to the batch-rerun UI.| v1.5.0    |
+|ML-4758|In rare cases, deleting a heavy project is unsuccessful and results in a timeout error message while the project moves to offline state.| Delete again.| v1.5.0    |
+|ML-4767|Torch 2.1.0 is not compatible with mlrun-gpu image.              | NA | v1.5.0 |
+|ML-4907|MLRun Client does not support Win OS.                                                 | NA | v1.3.0 | 
+|ML-4942|Dask dashboard does not work on managed clouds.|Dask is externally available thru NodePort, meaning ports needs to be manually open by the infrastructure provider. If running MLRun locally or CE, make sure to port-forward the port Dask Dashboard uses to ensure it is available externally to the Kubernetes cluster. | v1.5.0 |
```
Comment (gilad-shaham) [inline docs/change-log/index.md:748]: The description is too generic, the issue is that dask dashboard requires the relevant node ports to be open

File: docs/change-log/index.md [hunk 5]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-4992|Source target dir ownership is set to root. |Set target dir, and change workdir back:
```
Comment (gilad-shaham) [inline docs/change-log/index.md:749]: need more specific description for the issue

File: docs/change-log/index.md [hunk 6]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
 |---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|     
+|ML-1278|A user's permission to store data on the underlying storage depends on its V3IO permissions, not on project membership.                                                               |NA                                                               | v0.8.0
```
Comment (gilad-shaham) [inline docs/change-log/index.md:760]: When setting permissions to the project, the right permissions to the data storage need to be set in order to allow access to the data.

File: docs/change-log/index.md [hunk 7]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-3520|MLRun does not decompress large Kubeflow pipelines                                                                                                                                   |NA                                                                                                                                                                                                                                                                                                                              |v1.3.0    |
+|ML-3824|MLRun supports TensorFlow up to 2.11.                                                                                                                                                |NA                                                                                                                                                                                                                                                                                                                              |v1.3.1    |
```
Comment (gilad-shaham) [inline docs/change-log/index.md:764]: Please recheck, I think that part was addressed

File: docs/change-log/index.md [hunk 8]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
 |ML-3743|Setting AWS credentials as project secret cause a build failure on EKS configured with ECR.                                                                                          |When using an ECR as the external container registry, make sure that the project secrets AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY have read/write access to ECR, as described in the [platform documentation](https://www.iguazio.com/docs/latest-release/services/app-services/docker-registry/#create-off-cluster-registry)|
-|ML-4767|Use PyTorch versions up to and including than 2.0.1, but not higher.                                                                                                                  |
+|ML-4386|Local notifications aren’t persisted in mlrun API| NA | v1.5.0|
```
Comment (gilad-shaham) [inline docs/change-log/index.md:767]: local runs with notifications

File: docs/change-log/index.md [hunk 9]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-4767|Use PyTorch versions up to and including than 2.0.1, but not higher.       | NA |                 v1.5.0|
```
Comment (gilad-shaham) [inline docs/change-log/index.md:768]: this is with the mlrun-gpu image, not general, but please recheck

File: docs/change-log/index.md [hunk 10]
```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
 |ML-3743|Setting AWS credentials as project secret cause a build failure on EKS configured with ECR.                                                                                          |When using an ECR as the external container registry, make sure that the project secrets AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY have read/write access to ECR, as described in the [platform documentation](https://www.iguazio.com/docs/latest-release/services/app-services/docker-registry/#create-off-cluster-registry)|
-|ML-4767|Use PyTorch versions up to and including than 2.0.1, but not higher.                                                                                                                  |
+|ML-4386|Local notifications aren’t persisted in mlrun API| NA | v1.5.0|
+|ML-4767|Use PyTorch versions up to and including than 2.0.1, but not higher.       | NA |                 v1.5.0|   
+|ML-4960|The Memberships tab in Projects > Settings is part of MLRun only when integrated with the platform.            | NA |                 v1.5.0|
```
Comment (gilad-shaham) [inline docs/change-log/index.md:769]: The issue is with the caching, the fact that it's only part of the platform is not a limitation

**Review by gilad-shaham — CHANGES_REQUESTED** [review]

File: docs/change-log/index.md (hunk 2 above)
Comment (jillnogold) [reply docs/change-log/index.md:737]: moved to fixed in v1.4.0

File: docs/change-log/index.md [hunk 11]
```diff
@@ -198,6 +201,7 @@ See [Deprecations and removed code](#deprecations-and-removed-code).
 |ML-3508|Improved description of list_runs. See {py:class}`~mlrun.projects.MlrunProject.list_runs` [View in Git](https://github.com/mlrun/mlrun/pull/3686).                                                                                                                                                     |
 |ML-3621|`clear_context()` now does not delete content if the path is relative; and if a subpath exists, only the sub dir is deleted/cleared. [View in Git](https://github.com/mlrun/mlrun/pull/3689).                                                                                                          |
 |ML-3631 |MLRun now successfully pulls the source code from gitlab with a personal access token. [View in Git](https://github.com/mlrun/mlrun/pull/3927).                                                                                                                                                        |
+|ML-3633|Fail to import a context from dict. [View in Git](https://github.com/mlrun/mlrun/pull/3308).                                                                                                |
```
Comment (gilad-shaham) [inline docs/change-log/index.md:204]: If this is a fix we should say it is now possible to do it

**Review by gilad-shaham — CHANGES_REQUESTED** [review]

File: docs/change-log/index.md [hunk 12]
```diff
@@ -82,7 +82,10 @@ See [Deprecations and removed code](#deprecations-and-removed-code).
 ###  Closed issues
 | ID     |Description                                                                   |
 |---------|-------------------------------------------------------------------------------|
-|ML-4188|Deleting a project failed in the backend but was successfully deleted in UI.  |
+|ML-1584|Can now run `code_to_function` when filename contains special characters.|
+|ML-2199|Spark operator job doe not fail with default requests args.                   |
```
Comment (alonmr) [inline docs/change-log/index.md:86]:
```suggestion
|ML-2199|Spark operator job does not fail with default requests args.                   |
```

File: docs/change-log/index.md [hunk 13]
```diff
@@ -197,7 +200,8 @@ See [Deprecations and removed code](#deprecations-and-removed-code).
...
 |ML-3621|`clear_context()` now does not delete content if the path is relative; and if a subpath exists, only the sub dir is deleted/cleared. [View in Git](https://github.com/mlrun/mlrun/pull/3689).                                                                                                          |
-|ML-3631 |MLRun now successfully pulls the source code from gitlab with a personal access token. [View in Git](https://github.com/mlrun/mlrun/pull/3927).                                                                                                                                                        |
+|ML-3631|MLRun now successfully pulls the source code from gitlab with a personal access token. [View in Git](https://github.com/mlrun/mlrun/pull/3927).                                                                                                                                                        |
+|ML-3633|Can now import a context from dict. [View in Git](https://github.com/mlrun/mlrun/pull/3308).                                                                                                |
```
Comment (alonmr) [inline docs/change-log/index.md:204]:
```suggestion
|ML-3633|Fix date fields parsing when importing a context from dict. [View in Git](https://github.com/mlrun/mlrun/pull/3308).                                                                                                |
```

File: docs/change-log/index.md [hunk 14]
```diff
@@ -717,38 +721,56 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-4758|In rare cases, deleting a heavy project is unsuccessful and results in a timeout error message while the project moves to offline state.| Delete again.| v1.5.0    |
+|ML-4767|Torch 2.1.0 is not compatible with mlrun-gpu image.              | NA | v1.5.0 |
+|ML-4907|MLRun Client does not support Win OS.                                                 | Use WSL instead. | v1.3.0 | 
+|ML-4942|The Dask dashboard requires the relevant node ports to be open. |Your infrastructure provider must open the ports manually. If running MLRun locally or CE, make sure to port-forward the port Dask Dashboard uses to ensure it is available externally to the Kubernetes cluster. | v1.5.0 |
+|ML-4992|When a source archive is specified, the docker image's working directory is no longer automatically set to the target directory of that source archive.|Set target dir, and change workdir back: 
+   ```sj.with_source_archive(
+       source=project.source, pull_at_runtime=False, target_dir="/igz/mlrun_code"
+   )
+   sj.spec.build.extra = "WORKDIR /igz"
+   ```
```
Comment (alonmr) [inline docs/change-log/index.md:756]:
```suggestion
   ``` |v1.5.0|
```

File: docs/change-log/index.md [hunk 15]
```diff
@@ -717,38 +721,56 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+
 
 ## Limitations
-| ID     |Description                                                                                                                                                                          |Workaround                                                                                                                                                                                                                                                                                                                      |Opened in |
+| ID     |Desc                                                                                                                                |Workaround |Opened in|
```
Comment (alonmr) [inline docs/change-log/index.md:760]: ?
```suggestion
| ID     |Description                                                                                                                                |Workaround |Opened in|
```

File: docs/change-log/index.md [hunk 16]
```diff
@@ -717,38 +721,56 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
 |ML-3743|Setting AWS credentials as project secret cause a build failure on EKS configured with ECR.                                                                                          |When using an ECR as the external container registry, make sure that the project secrets AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY have read/write access to ECR, as described in the [platform documentation](https://www.iguazio.com/docs/latest-release/services/app-services/docker-registry/#create-off-cluster-registry)|
-|ML-4767|Use PyTorch versions up to and including than 2.0.1, but not higher.                                                                                                                  |
+|ML-4386|Notifications of local runs aren't persisted. | NA | v1.5.0|
+|ML-4767|When using mlrun-gpu image, use PyTorch versions up to and including than 2.0.1, but not higher.       | NA |                 v1.5.0|   
+|ML-4855|MLRun supports TensorFlow up to 2.13.1.
```
Comment (alonmr) [inline docs/change-log/index.md:771]: this is duplicated with ML-3824

**Review by alonmr — COMMENTED** [review]
Minor comments

**Pull request merged**